default.env

```diff
@@ -0,0 +1,6 @@
+DATABASE_URL=postgresql://users_api:users_api@localhost/users_api
+TEST_DATABASE_URL=postgresql://users_api:users_api@localhost/test_users_api
+# openssl rand -hex 32
+SECRET_KEY=
```

Comment:
Is this something that the user needs to set before testing, or is it ok to have an empty string default for `SECRET_KEY`?

Reply:
it can be a random string, but that command generates a more secure/random string with larger size